pynvml/nvml.py

```diff
@@ -1245,6 +1245,13 @@ def nvmlDeviceGetPowerUsage(handle):
     check_return(ret)
     return c_watts.value
 
+def nvmlDeviceGetTotalEnergyConsumption(handle):
+    c_joules = c_uint()
```

Comment:
you could call this variable `c_mJoules` if it is in mJ :)

Reply:
good catch :)

Reply:
I was following the examples set by other parts of the code where for instance `c_watts` is used instead of `c_mWatts`. Can I make such changes to other parts of the code here, or do you prefer cleanup commits unrelated to some PR to be made in a separate PR?

Reply:
> Can I make such changes to other parts of the code here, or do you prefer cleanup commits unrelated to some PR be made in a separate PR?

Either way seems fine to me.  It is up to you if you want to make the other changes outside of `nvmlDeviceGetTotalEnergyConsumption` in this PR

Reply:
Done as a separate commit in this PR.